projects/mysql-server/build.sh

```diff
@@ -27,6 +27,8 @@ else
 fi
 make install
 cp $OUT/mysql/bin/fuzz* $OUT/
+#temporary disabled
```

Comment:
"# FIXME: Temporary disabled due to ...." and end comment with period.

Reply:
Ok